core/frontend/src/libs/MAVLink2Rest/index.ts

```diff
@@ -13,6 +13,9 @@ import Listener from './Listener'
 const messageId: Dictionary<number> = {
   HEARTBEAT: 0,
   SYS_STATUS: 1,
+  SCALED_PRESSURE: 29,
+  SCALED_PRESSURE2: 137,
+  VFR_HUD: 74,
 }
```

Comment:
I just created https://github.com/bluerobotics/BlueOS-docker/pull/1393

Reply:
Note that if #1393 gets merged in before this PR, the error message on line 146 of this file would need to change which file it specified to add the new mapping to.